src/probability/ident_distrib.lean

```diff
@@ -283,4 +290,58 @@ end
 
 end ident_distrib
 
+section uniform_integrable
+
+variables {E : Type*} [measurable_space E] [normed_add_comm_group E] [borel_space E]
+  {μ : measure α} [is_finite_measure μ]
+
+/-- This lemma is superceded by `integrable.uniform_integrable_of_ident_distrib` which only require
+`ae_strongly_measurable`. -/
+lemma integrable.uniform_integrable_of_ident_distrib' {ι : Type*} {f : ι → α → E}
```

Comment:
you are doing the L^1 version here, but can you do instead the general L^p version (which shouldn't be harder)? Then the strong law of large numbers in L^p would follow directly.